packages/flutter/lib/src/gestures/tap_and_drag.dart

```diff
@@ -534,6 +501,14 @@ mixin _TapStatusTrackerMixin on OneSequenceGestureRecognizer {
   Timer? _consecutiveTapTimer;
   Offset? _lastTapOffset;
 
+  // Callback used to indicate that a tap tracking has started.
+  VoidCallback? onTapTrackStart;
```

Comment:
This is cool! I just learned that mixins add instance fields to the class they are mixed into. :D

Reply:
How does this work for dart-docs? Ideally documentation for `onTapTrackStart` and `onTapTrackReset` would also show up in the api docs for `TapAndDragGestureRecognizer`.

Reply:
Good catch - I'll convert these to ///. I can see they display nicely when hovering over `TapAndDragGestureRecognizer.onTapTrack*`.

Reply:
I wonder if we should also remove `maxConsecutiveTap` from `TapAndDragGestureRecognizer` since the mixin already adds the instance fields. We would probably have to change `int? get maxConsecutiveTap` to `int? maxConsecutiveTap`.

My only concern is that these members are in a private class and public docs are being kept in there, do you have any thoughts on this @Hixie?